submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <title></title>
+    <link rel="stylesheet" href="./stylesheet/main.css" />
+    <link rel="stylesheet" href="./stylesheet/header.css" />
+    <link rel="stylesheet" href="./stylesheet/choose-document.css" />
+    <link rel="stylesheet" href="./stylesheet/upload.css" />
+    <link rel="stylesheet" href="./stylesheet/language-choose.css" />
+    <link rel="stylesheet" href="./stylesheet/tone-time.css" />
+    <link rel="stylesheet" href="./stylesheet/payment.css" />
+    <link rel="stylesheet" href="./stylesheet/footer.css" />
+    <link rel="stylesheet" href="./stylesheet/normalize.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <div class="innerHeader">
+        <button class="headerButtons">Back</button>
+        <a href="#"><img class="header" src="img/logo-header.png"/></a>
+        <button class="headerButtons">Sign in</button>
+      </div>
+    </header>
+    <main class="adjust">
+      <div class="container">
+        <form class="" action="index.html" method="post">
+          <div class="upperMenu">
+            <div class="orderInfo">
+              <div class="left">
+                <div class="summary">
+                  <h3 class="summaryHeader">YOUR ORDER</h3>
+                  <table class="wideTable">
+                    <tbody>
+                      <tr>
+                        <th class="alignTh">Items</th>
+                        <th class="alignTh">Details</th>
+                        <th class="alignTh">Price</th>
+                        <th class="alignTh">Delivery</th>
+                      </tr>
+                      <tr>
+                        <td><b>2</b> documents</td>
+                        <td><b>2</b> languages</td>
+                        <td>$ <b>696.06</b></td>
+                        <td><b>48</b> documents*</td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              </div>
+              <div class="right">
+                <button class="upperMenuButton order">PLACE ORDER</button>
+                <button class="upperMenuButton">PRINT A QUOTE</button>
```

Comment:
Print button shoud not submit your form. Use `type='button'` for this one.
Don't use text in uppercase as content. Use CSS for this. When you change text your visual representation should be the same and independent of text content or text case inside HTML.

Reply:
Done!